plugins/node/opentelemetry-plugin-mysql/src/utils.ts

```diff
@@ -87,3 +68,23 @@ function getJDBCString(
 
   return jdbcString;
 }
+
+/**
+ * Conjures up the value for the db.statement attribute by formatting a SQL query.
+ *
+ * @returns the database statement being executed.
+ */
+export function getDbStatement(
+  query: string | Query | QueryOptions,
+  format: Function,
+  values?: any[]
+) {
```

Comment:
1. return type is missing
2. What if query is not a string and not a query, should this still return something ?

Reply:
Thanks for spotting this @obecny , I fixed `format`'s type, as well as the function's logic, as to always ensure a string is returned.

Reply:
@naseemkullah can you please check again the point 1 - the return type of this function is still missing - line 86

Reply:
Oh I thought inferred type would suffice, I will go ahead an make it explicit